doozerlib/constants.py

```diff
@@ -6,3 +6,5 @@
     "GIT_SSH_COMMAND": "ssh -oBatchMode=yes",
     "GIT_TERMINAL_PROMPT": "0",
 }
+
+BREW_HUB = "https://brewhub.engineering.redhat.com/brewhub"
```

Comment:
we have this configured in group.yml, could it use that please?

Reply:
Sure. updated.